ui/app/AppLayouts/Chat/views/ActivityCenterMessageComponentView.qml

```diff
@@ -140,18 +139,11 @@ Item {
                 }
 
                 activityCenter.close()
-
-//                if (model.message.communityId) {
-//                    root.chatSectionModule..communities.setActiveCommunity(model.message.communityId)
-//                }
-
-//                root.chatSectionModule.setActiveChannel(model.chatId);
-
-                // To position to appropriate message check how it's done in AppSearch module,
-                // similar may be done here, corresponding module for activity center just need to
-                // use the mechanism done there, send the signal form message service and
-                // and the rest is already handled.
-//                positionAtMessage(model.id);
+                var sectionId = "chat"
```

Comment:
There is probably a Constant for that

Reply:
yes, only in nim, well that is all I could find